l10n_eu_nace/migrations/12.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def migrate(cr, version):
+    cr.execute(
+        """
+        UPDATE ir_model_data
+        SET name=concat('old_', name)
```

Comment:
Without this patch, possibly this data would be deleted as the module gets updated, and the post migration script would fail:

```suggestion
        SET name=concat('old_', name), noupdate=1
```